### PR TITLE
Adds a note about controllership and and ownership of the identifier

### PR DESCRIPTION
**Pull request by kdenhartog**
This is a first attempt at addressing issue #269

Signed-off-by: Kyle Den Hartog <kyle.denhartog@mattr.global>

@jandrieu can you give it a read to see if it sums out point 1 well enough?

***
<a href="https://pr-preview.s3.amazonaws.com/kdenhartog/did-core/pull/352.html" title="Last updated on Aug 12, 2020, 2:43 AM UTC (3e73b58)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/352/78f1148...kdenhartog:3e73b58.html" title="Last updated on Aug 12, 2020, 2:43 AM UTC (3e73b58)">Diff</a>

### CODE REVIEW

**Comment by Fak3** [comment]
Could you please add linebreaks to make a readable raw file?

File: index.html [hunk 1]
```diff
@@ -3555,6 +3555,21 @@ <h3>
...
 specifications SHOULD explain and specify their implementation if applicable.
         </p>
+
+        <div class="note" role="note" id="issue-container-generatedID-8">
+          <div role="heading" class="note-title marker" id="h-note-7" 
+          aria-level="4"><span>Note</span></div><p class="">
+Through the use of digital signatures a controller can assert proof of control 
+which can be used in a variety of different contexts. By asserting proof of 
+control of the identifier, the controller is asserting proof of ownership of 
+the public key which may not be the same as the subject of the DID. However,
```
Comment (Fak3) [inline index.html:3565]: Public key is never subject of a DID. Is subject of a DID actually related in any way to the proof of control?

Comment (kdenhartog) [reply index.html:3565]: Yes, the controller (who needs to prove control - usually with a digital signature) has the authority to assert who the DID identifies as the subject. @jandrieu comments in #269 goes into better detail about the relationship between the two.

Comment (Fak3) [reply index.html:3565]: This particular statement is confusing: "public key may not be the same as the subject of the DID." It does not really help to clarify the relationship between subject and controller. Public key and a thing identified by a DID are completely different entities. Perhaps you wanted to say that subject of a did may or may not have a control over did?

Comment (kdenhartog) [reply index.html:3565]: Ahhh I see what you're pointing out now. I'll give an attempt to reword this a bit based on the way you put this.

File: index.html [hunk 2]
```diff
@@ -3555,6 +3555,21 @@ <h3>
...
         </p>
+
+        <div class="note" role="note" id="issue-container-generatedID-8">
+          <div role="heading" class="note-title marker" id="h-note-7" 
+          aria-level="4"><span>Note</span></div><p class="">
+Through the use of digital signatures a controller can assert proof of control 
+which can be used in a variety of different contexts. By asserting proof of 
+control of the identifier, the controller is asserting proof of ownership of 
+the public key which may not be the same as the subject of the DID. However, 
+it's up to the verifier to determine if the usage of digital signatures is
```
Comment (msporny) [inline index.html:3566]: Avoid use of contractions in official specifications; use more formal language. If you find yourself using a large number of contractions, it is typically a sign that your sentence needs to be reworded to be simpler.

File: index.html [hunk 3]
```diff
@@ -3555,6 +3555,21 @@ <h3>
 Signatures on <a>DID documents</a> are optional. <a>DID method</a>
 specifications SHOULD explain and specify their implementation if applicable.
         </p>
+
+        <div class="note" role="note" id="issue-container-generatedID-8">
+          <div role="heading" class="note-title marker" id="h-note-7" 
+          aria-level="4"><span>Note</span></div><p class="">
+Through the use of digital signatures a controller can assert proof of control
```
Comment (msporny) [inline index.html:3562]: "proof of control" needs to be defined somewhere... it's used a number of times in the specification without really going into what "proof of control" and "proof of ownership" mean... we need to be explicit about what we mean. It is effectively "Using private cryptographic material to generate a digital signature to prove that one is in control of the private cryptographic material."

File: index.html (hunk 2 above)
Comment (jandrieu) [reply index.html:3566]: I would remove the language that proof of control is related to "proof of ownership". A major point of the language of "proof-of-control" is that ownership is a matter of law, not technology. So, only courts can establish actual ownership. Proof-of-control, on the other hand, is technically concrete. So, when someone steals your keys, they *can* generate proof-of-control, but they CANNOT establish proof of ownership. "Not your keys, not your coins" fails in the reverse: "Controlling the keys does not prove they are your coins." You *can* spend them--you have control--but you don't own them any more than the car thief who stole your keys to steal your car. They don't "own" your car, nor do they establish proof of ownership. They just control it.

**Comment by msporny** [comment]
Still waiting on that rewording @kdenhartog ... ETA on getting that done?

**Comment by kdenhartog** [comment]
Sorry was away on vacation for a bit and didn't get back to this until now. I made an attempt to cleanup the language based on this discussion. Please review and see if it clarifies better the intent.

>"proof of control" needs to be defined somewhere

I agree that this should be defined given the prevelance of usage in the specification, but it seems to fall a bit out of scope of what I was trying to get at with this issue/PR. Can we add that separately in a different PR to keep this PR small and focused?

**Comment by kdenhartog** [comment]
In light of #373 I'm considering closing this PR and closing the issue because #373 describes a more generalized view of the problem with far greater description than this single note could do justice. Thoughts on that approach @msporny and @jandrieu ?

**Comment by msporny** [comment]
> In light of #373 I'm considering closing this PR and closing the issue because #373 describes a more generalized view of the problem with far greater description than this single note could do justice. Thoughts on that approach @msporny and @jandrieu ?

Up to you @kdenhartog ... those appendices have changed over the past several weeks of discussion. I do think they're probably going to cover what you have in this PR to some degree. You may want to keep this PR open until you feel like it's dealt with in #373. Again, your call.

**Comment by msporny** [comment]
Looking for feedback here, @kdenhartog -- it's been 27 days. I'm going to mark this pending close if you don't engage in the next 7 days.

**Comment by kdenhartog** [comment]
Looks like the original comment 13 days ago in the issue should have been cross posted here as well:

> The PR exists for this issue, but I'm in favor of closing it and the PR #352 once the appendix work being discussed in #373 ends up in the spec or as an additional side document. Either would satisfy my question for this issue.

I've been waiting to see the results of #373 . As far as I can tell the PRs for those are still in progress and if they're only discussing additional properties then I would want this PR to go in. However, if the PRs will add the aspects about the relationship of the DID Document, the Controller, and the DID Subject then I'm in favor of closing this.

The notes on [8/13](https://www.w3.org/2019/did-wg/Meetings/Minutes/2020-08-13-did-topic) left it at more discussion on the topic is needed with the closest point to consensus being some would end up in the spec and some would end up in a note. The notes on [8/25](https://www.w3.org/2019/did-wg/Meetings/Minutes/2020-08-25-did) seem to imply we'll be adding a `type` property back which should help in identifying a DID Subject if it's an information resource. However it doesn't go into further detail about the relationship between the 3. Looking at the comments on the thread and the other meeting notes after 8/23, I see no further mention of how we're going to further document the relationship between the 3.

So at this point, I'm waiting to see what goes into the DID Spec and what won't so I can decide if it's necessary to continue this PR to document that relationship or see if it's going to be covered in alternative PRs. Would you prefer that I add these aspects myself in this PR and then potentially have them stated elsewhere as well or would you prefer that this PR and issue wait until we see the end result of #373 ?

I'm happy with either approach.

**Comment by msporny** [comment]
> Would you prefer that I add these aspects myself in this PR and then potentially have them stated elsewhere as well or would you prefer that this PR and issue wait until we see the end result of #373 ?

Neither :) -- I'd like you to work with @talltree to get the appendix in. Letting PRs like this ride w/ no end in sight isn't the way we're supposed to be using PRs. Either it's focused and the PR goes in relatively quickly (within a month a most), or it's not, it hasn't made progress, and it's closed.

This PR seems to potentially conflict with @talltree's proposed appendices, so I'd rather you work with him on his PR... OR, you write the appendix PR. In other words - I'd like to see you do active work to move the PR forward that the group seems to have consensus on (@talltree's PR), merge this language into that PR, and drop this PR.

Thoughts?

**Comment by kdenhartog** [comment]
> > Would you prefer that I add these aspects myself in this PR and then potentially have them stated elsewhere as well or would you prefer that this PR and issue wait until we see the end result of #373 ?
> 
> Neither :) -- I'd like you to work with @talltree to get the appendix in. Letting PRs like this ride w/ no end in sight isn't the way we're supposed to be using PRs. Either it's focused and the PR goes in relatively quickly (within a month a most), or it's not, it hasn't made progress, and it's closed.
> 
> This PR seems to potentially conflict with @talltree's proposed appendices, so I'd rather you work with him on his PR... OR, you write the appendix PR. In other words - I'd like to see you do active work to move the PR forward that the group seems to have consensus on (@talltree's PR), merge this language into that PR, and drop this PR.
> 
> Thoughts?

I'll go ahead and close this PR and check in with @talltree to see where things are at with the appendix work.

**Pull request closed without merging**